weave/logs.py

```diff
@@ -227,8 +232,8 @@ def configure_logger() -> None:
             # terminal, you get the logs.
             enable_stream_logging(
                 logger,
-                wsgi_stream_settings=LogSettings(LogFormat.PRETTY, "INFO"),
-                pid_logfile_settings=LogSettings(LogFormat.PRETTY, "INFO"),
+                wsgi_stream_settings=LogSettings(LogFormat.PRETTY, level=None),
```

Comment:
The explicit `logging.INFO` level here meant that debug logs would not show in the dev server even when `WEAVE_LOG_LEVEL=DEBUG`.